packages/puppeteer-core/src/api/HTTPResponse.ts

```diff
@@ -89,14 +89,14 @@ export abstract class HTTPResponse {
    * failed to detect the correct encoding, the buffer might
    * be encoded incorrectly. See https://github.com/puppeteer/puppeteer/issues/6478.
    */
-  abstract buffer(): Promise<Buffer>;
+  abstract buffer(): Promise<Uint8Array>;
```

Comment:
so looking at this diff, I am actually not sure we should remove Buffer from this API (since the getter's name is buffer). I wonder if we can keep it for Node users as it seems convenient but implement a method `array(): Promise<Uint8Array>` in addition. WDYT?

Reply:
I will only start looking at such things once everything is passing, but it was also on my mind.